runtime/jcl/exports.cmake

```diff
@@ -695,73 +695,261 @@ if(J9VM_OPT_JFR)
+			Java_jdk_jfr_internal_JVM_beginRecording
+			Java_jdk_jfr_internal_JVM_commit
+			Java_jdk_jfr_internal_JVM_counterTime
+			Java_jdk_jfr_internal_JVM_createJFR
+			Java_jdk_jfr_internal_JVM_destroyJFR
+			Java_jdk_jfr_internal_JVM_emitEvent
+			Java_jdk_jfr_internal_JVM_emitOldObjectSamples
+			Java_jdk_jfr_internal_JVM_endRecording
+			Java_jdk_jfr_internal_JVM_exclude
+			Java_jdk_jfr_internal_JVM_flush__
+			Java_jdk_jfr_internal_JVM_flush__Ljdk_jfr_internal_event_EventWriter_2II
+			Java_jdk_jfr_internal_JVM_getAllEventClasses
+			Java_jdk_jfr_internal_JVM_getAllowedToDoEventRetransforms
+			Java_jdk_jfr_internal_JVM_getChunkStartNanos
+			Java_jdk_jfr_internal_JVM_getClassId
+			Java_jdk_jfr_internal_JVM_getConfiguration
+			Java_jdk_jfr_internal_JVM_getDumpPath
+			Java_jdk_jfr_internal_JVM_getEventWriter
+			Java_jdk_jfr_internal_JVM_getPid
+			Java_jdk_jfr_internal_JVM_getStackTraceId
+			Java_jdk_jfr_internal_JVM_getThreadId
+			Java_jdk_jfr_internal_JVM_getTicksFrequency
+			Java_jdk_jfr_internal_JVM_getTimeConversionFactor
+			Java_jdk_jfr_internal_JVM_getTypeId__Ljava_lang_Class_2
+			Java_jdk_jfr_internal_JVM_getTypeId__Ljava_lang_String_2
+			Java_jdk_jfr_internal_JVM_getUnloadedEventClassCount
+			Java_jdk_jfr_internal_JVM_hostTotalMemory
+			Java_jdk_jfr_internal_JVM_include
+			Java_jdk_jfr_internal_JVM_isAvailable
+			Java_jdk_jfr_internal_JVM_isContainerized
+			Java_jdk_jfr_internal_JVM_isExcluded__Ljava_lang_Class_2
+			Java_jdk_jfr_internal_JVM_isExcluded__Ljava_lang_Thread_2
+			Java_jdk_jfr_internal_JVM_isInstrumented
+			Java_jdk_jfr_internal_JVM_isRecording
+			Java_jdk_jfr_internal_JVM_log
+			Java_jdk_jfr_internal_JVM_logEvent
+			Java_jdk_jfr_internal_JVM_markChunkFinal
+			Java_jdk_jfr_internal_JVM_newEventWriter
+			Java_jdk_jfr_internal_JVM_registerNatives
+			Java_jdk_jfr_internal_JVM_retransformClasses
+			Java_jdk_jfr_internal_JVM_setCompressedIntegers
+			Java_jdk_jfr_internal_JVM_setConfiguration
+			Java_jdk_jfr_internal_JVM_setCutoff
+			Java_jdk_jfr_internal_JVM_setDumpPath
+			Java_jdk_jfr_internal_JVM_setEnabled
+			Java_jdk_jfr_internal_JVM_setFileNotification
+			Java_jdk_jfr_internal_JVM_setForceInstrumentation
+			Java_jdk_jfr_internal_JVM_setGlobalBufferCount
+			Java_jdk_jfr_internal_JVM_setGlobalBufferSize
+			Java_jdk_jfr_internal_JVM_setMemorySize
+			Java_jdk_jfr_internal_JVM_setMethodSamplingPeriod
+			Java_jdk_jfr_internal_JVM_setOutput
+			Java_jdk_jfr_internal_JVM_setRepositoryLocation
+			Java_jdk_jfr_internal_JVM_setStackDepth
+			Java_jdk_jfr_internal_JVM_setStackTraceEnabled
+			Java_jdk_jfr_internal_JVM_setThreadBufferSize
+			Java_jdk_jfr_internal_JVM_setThreshold
+			Java_jdk_jfr_internal_JVM_setThrottle
+			Java_jdk_jfr_internal_JVM_shouldRotateDisk
+			Java_jdk_jfr_internal_JVM_storeMetadataDescriptor
+			Java_jdk_jfr_internal_JVM_subscribeLogLevel
+			Java_jdk_jfr_internal_JVM_uncaughtException
+			java_jdk_jfr_internal_JVM_emitDataLoss
```

Comment:
Please keep the list of exports sorted; this belongs before line 820 (`Java_jdk_jfr_internal_JVM_emitEvent`).